app/forms/household_relationship_form.py

```diff
@@ -71,12 +71,10 @@ def answer_errors(self, input_id):
             return [error[1] for error in self.map_errors() if input_id == error[0]]
 
     answer = SchemaHelper.get_first_answer_for_block(block_json)
-    guidance = answer['guidance'] if 'guidance' in answer else ''
-    label = answer['label'] if 'label' in answer else ''
 
     field = FieldList(SelectField(
-        label=label,
-        description=guidance,
+        label=answer.get('guidance'),
```

Comment:
Note that you'll get None returned by default now where before it was `''`; is that an intentional change, if not add the default argument to `get`.